README.md

```diff
@@ -3,4 +3,48 @@
 [![Travis](https://img.shields.io/travis/findologic/libflexport.svg)](https://travis-ci.org/findologic/libflexport)
 [![Latest Stable Version](https://poser.pugx.org/findologic/libflexport/v/stable)](https://packagist.org/packages/findologic/libflexport)
 
-**Highly experimental! Don't use it!**
\ No newline at end of file
+## Synopsis
+
+This project provides a export library for XML and CSV generation according to the FINDOLOGIC export patterns.
+* [XML export patterns](https://docs.findologic.com/doku.php?id=export_patterns:xml)
```

Comment:
Links to the documentation pages should be directly when mentioning `XML` and `CSV`.

Reply:
Done